src/openloco/game_commands.h

```diff
@@ -19,10 +19,20 @@ namespace openloco::game_commands
         flag_6 = 1 << 6, // 0x40
     };
 
+    enum class GameCommand : uint8_t
```

Comment:
I really like this. While beyond the scope of this PR, we should really add the others as well. (I previously added comments to all game command callers (`do_`).)

Reply:
yeah we were discussing the other game commands and realised we had a bit of knowledge not written down anywhere.